src/main/java/com/devonfw/ide/sonarqube/common/impl/SonarDevon4jPlugin.java

```diff
@@ -17,7 +17,7 @@
 
   private static final String DISABLED = "Disabled";
 
-  private static final String QUALINSIGHT = "qualinsight-plugins-sonarqube-smell-plugin";
+  private static final String QUALINSIGHT = "qualinsight-sonarqube-smell-plugin";
```

Comment:
was this wrong from the start or has that plugin be renamed and the name might depend on the version of SonarQube server?

Reply:
The naming of this plugin is a little inconsistent... Most of the released versions are using `qualinsight-sonarqube-smell-plugin', except for 3.0.0 & 2.0.0-RC2 (see [releases](https://github.com/QualInsight/qualinsight-plugins-sonarqube-smell/releases))

Reply:
Thanks for that explanation. It might have been better then to implement some magic in our plugin that will try both names and see if one of them works. We might break existing installations using the old name with this change...